### ISSUE TITLE
porous iris

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9766]
**Preferred term label:**
Porous iris
This may be related to HP:0011523 or HP:0500007 but does not seem to be included in those existing terms.

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Oculodentodigital dysplasia (Smith's Malformation textbook describes "fine porous iris")

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I am wondering if you have a picture or a more detailed definition.
The only thing I can find is 
"fine porous spongy iris abnormalities"
from
https://www.cell.com/ajhg/fulltext/S0002-9297(07)60549-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0002929707605497%3Fshowall%3Dtrue
but this is not really useful because as far as I can see subsequent articles did not note this and it is unclear exactly how an ophthalmologist would recognize this finding.

**Comment by MickeySegal** [comment]
All I could find other than Smith is https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4869463/
"Ophthalmic findings include microphthalmia, microcornea, fine porous spongy iris abnormalities, cataracts, glaucoma and optic atrophy" and the original PDF of the Parashari et al. reference linked at https://www.ajol.info/index.php/sajr/article/view/73485 just says "Common ocular features with this condition are microphthalmia, microcornea, fine porous spongy iris abnormalities, cataracts, glaucoma, optic atrophy and other eye abnormalities that can lead to vision loss."

It may be best to delete the finding.

**Comment by pnrobinson** [comment]
Let's delete.
